index.js

```diff
@@ -608,6 +794,24 @@
     'ninth'
   ];
 
+  //  ordinal :: Integer -> String
+  var ordinal = function(idx) {
+    return ordinals[idx];
+  };
+
+  //  ordinalList :: [Integer] -> String
+  var ordinalList = function(indexes) {
+    if (indexes.length === 1) {
+      return ordinals[indexes[0]];
+    } else {
+      var init = indexes.slice(0, -1);
+      return (
+        map(init, ordinal).join(', ') + ' and ' +
+        ordinals[indexes[indexes.length - 1]]
+      );
+    }
+  };
+
```

Comment:
Here's a slightly improved version:

```javascript
//  ordinalList :: [Integer] -> String
var ordinalList = function(indexes) {
  var words = map(indexes, function(index) { return ordinals[index]; });
  return words.length > 2 ?
    words.slice(0, -1).join(', ') + ', and ' + words[words.length - 1] :
    words.join(' and ');
};
```

This gives us my beloved Oxford comma. :)